docs/faq.md

```diff
@@ -6,34 +6,9 @@ nav_order: 4
 
 ## How do users include their own packages/binaries in a custom "bootc compatible" container?
 
-The "bootc compatible" containers are OCI container images, so you can customize them in the same way you build containers today.
+The "bootc compatible" containers are OCI container images, so you can customize them in the same way you build containers today. This means using a Containerfile to customize your image and build tools like `buildah`, `podman build`, or `docker build` to generate your customized "bootc compatible" container image.
 
-For example, using your own yum/dnf repo in a Dockerfile:
-
-```Dockerfile
-FROM quay.io/redhat/rhel-base:10
-COPY custom.repo /etc/yum.repos.d/
-RUN dnf -y install custom-rpm & \
-    dnf clean all && \
-    ostree container commit
-```
-
-Or using multi-stage builds in your Dockerfile:
-
-```Dockerfile
-# Build a small Go program
-FROM registry.access.redhat.com/ubi8/ubi:latest as builder
-WORKDIR /build
-COPY . .
-RUN yum -y install go-toolset
-RUN go build hello-world.go
-
-FROM quay.io/redhat/rhel-base:10
-COPY --from=builder /build/hello-world /usr/bin
-RUN ostree container commit
-```
-
-You can find more examples at the [centos-boot-layered repo](https://github.com/CentOS/centos-boot-layered) repo or the [CoreOS layering-examples repo](https://github.com/coreos/layering-examples).
+For examples of how use "bootc compatible" container images, see the [centos-boot-layered repo](https://github.com/CentOS/centos-boot-layered) or the [CoreOS layering-examples repo](https://github.com/coreos/layering-examples).
```

Comment:
Ultimately there are going to be multiple distributions using bootc, right?  So a single source-of-truth gets hard.

Now, centos-boot-layered is currently intended for images pre-built by the project that are not "base images".

One immediately obvious problem with trying to unify coreos/layering-examples with centos-boot stuff is that coreos/layering-examples uses rpm-ostree because there is no `dnf` in FCOS today.

Reply:
Fair point.  Let me rephrase.

Assuming I'd be a new user, I prefer to follow one known-to-work workflow.  Being pointed to multiple alternatives may confuse me.  (Based on previous experience,  I am implying that the docs' qualities vary and may not be up to date)

Reply:
What I desire is _one_ known-to-work example.  Pointing to multiple distros and implementations after that point is totally fine.

Reply:
How about I modify the text to point to `centos-boot` as an example of building a base image and point to `centos-boot-layered` as an example of derived images?

I feel those are the best options at the moment.

Reply:
Sounds great, thanks @miabbott !